src/lib/sql/sql_translator.hpp

```diff
@@ -102,6 +102,9 @@ class SQLTranslator final : public Noncopyable {
 
   std::shared_ptr<AbstractLQPNode> _translate_drop(const hsql::DropStatement& update);
 
+  std::shared_ptr<AbstractLQPNode> _apply_column_renames(const std::shared_ptr<AbstractLQPNode> node,
```

Comment:
This is a bit misleading. Other methods also rename columns, for example `_translate_aggregate`. If this is only used for subselects, maybe it should be named `_apply_subselect_alias` or sth like that